### DISCUSSION TITLE
What Stuff i have to know before i contribute ??

### DISCUSSION THREAD

**Discussion by Prathamix77** [TEAMMATES/teammates#12636] [Q&A] [answered]
I wanna know the tech stack and also the resources , I need to learn before helping up the project
PS : I really wanna contribute into this project

THANK YOU :)

**Comment by domlimm** [comment, marked as answer]
Hey @Prathamix77! Thank you for your interest in the project! For starters, it's always good to look at the documentation provided or the README.

Here's the link to the project's Contributing guidelines - https://teammates.github.io/teammates/contributing-doc.html.

Hope this clarifies and do let us know if you have any questions.
